.eslintignore

```diff
@@ -1 +0,0 @@
-dist
```

Comment:
Removing this causes issues when we run eslint, it considers files from `dist` folder, i think we should add back this file back

Reply:
I did like this instead https://github.com/snapshot-labs/snapshot-strategies/pull/706/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R22

Reply:
Do you know how to solve these?
![image](https://user-images.githubusercontent.com/15967809/178214418-caa48f9b-3a08-4242-9e76-8918bcf76a10.png)

Or we should disable this rule?